### ISSUE TITLE
centos 8 : redis and memcached not installed

### ISSUE THREAD

**Issue by franciscopaniskaseker** [litespeedtech/cyberpanel#448]
Yesterday I installed a centos 8 fresh server. I asked to setup script to install redis and memcached, but both services have not been installed.

memcached I installed manually:

```bash
dnf install memcached
systemctl enable --now memcached
```

But I am not sure if redis need to be something more than just install the service.

**Comment by qtwrk** [comment]
Hi,

I just run a test, which I did see redis not installed, but memcached installed.

for memcached, you may need to disable UDP.

could you please try reproduce it again  ? 

Best regards,

**Comment by franciscopaniskaseker** [comment]
In your setup the epel-release was installed? I had to install manually.

**Comment by qtwrk** [comment]
yes, epel-release was one of the very first package installed by installer

**Comment by domaingood** [comment]
+1

**Comment by AmiOta** [comment]
+1

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
